### ISSUE TITLE
INPUT-OUTPUT gets parsed as an error

### ISSUE THREAD

**Issue by gustason** [BalticAmadeus/AblFormatter#269]
For example, in the code:
``` abl
class MyEntityClass:
    method public void Fetch(input-output dataset dsMyDataset,
                             poParameter as myParameter):
    
    end method.
end class.
```

![image](https://github.com/user-attachments/assets/c9cfc2ec-a396-47fe-85f6-dfab184bbf9b)

It would be advantageous if `INPUT-OUTPUT` would be added as a `function_parameter_mode` node.

**Comment by eglekaz** [comment]
Registered: https://github.com/usagi-coffee/tree-sitter-abl/issues/99

**Comment by eglekaz** [comment]
Fixed with version 0.0.45 of tree-sitter-abl
